src/Microsoft.Diagnostics.EventFlow/Microsoft.Diagnostics.EventFlow.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could you please rename this project to `Microsoft.Diagnostics.EventFlow.Json`? It will reflect its purpose better.